lib/formatters.js

```diff
@@ -73,9 +74,35 @@ const serialize = (obj) => {
         .join(',');
 };
 
+function flatten(data, prefix = 'data') {
+    if (data && typeof data === 'object' && !Array.isArray(data)) {
+        let returnVal = {}
+        Object.keys(data).forEach(key => {
+            let val = data[key]
+            returnVal = Object.assign(returnVal, flatten(val, `${prefix}.${key}`))
+        })
+        return returnVal
+    } else {
```

Comment:
Do you need `else` here?

Reply:
I like it for the purpose of being explicit. Technically the way things are set up here currently I could omit the `else`, but I feel like it makes more sense to leave it for clarity.